vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/pool/SqlConnectionPool.java

```diff
@@ -394,5 +404,18 @@ public int getSecretKey() {
     public Connection unwrap() {
       return conn;
     }
+
+    public boolean hasIdleExpired(long now) {
+      return idleEvictionTimestamp < now;
+    }
+
+    public boolean hasLifetimeExpired(long now) {
+      return lifetimeEvictionTimestamp < now;
+    }
+
+    public boolean shouldEvict(long now) {
+      return hasIdleExpired(now) || hasLifetimeExpired(now);
+    }
```

Comment:
```suggestion
    private boolean hasIdleExpired(long now) {
      return idleEvictionTimestamp < now;
    }

    private boolean hasLifetimeExpired(long now) {
      return lifetimeEvictionTimestamp < now;
    }

    private boolean shouldEvict(long now) {
      return hasIdleExpired(now) || hasLifetimeExpired(now);
    }
```

Making these private will make the JIT's job easier (good candidates for inlining).

Reply:
done